docusaurus.config.js

```diff
@@ -88,6 +88,16 @@ const config = {
           target: "_self",
         },
         items: [
+          {
+            href: "/services/sdk/overview?sdk=ts",
+            label: "Typescript SDK",
```

Comment:
Nitpick – we have somewhat inconsistent capitalisation of TypeScript; I believe the canonical spelling is with a capital S :-)

```suggestion
            label: "TypeScript SDK",
```

Reply:
Also: should we refer to this as the TypeScript / JavaScript SDK? 🤔

Reply:
Oh you're right. I'll adapt that everywhere... Maybe in a separate PR to avoid overflowing this one.